src/Acme/Bundle/AppBundle/Model/ColorProductValue.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Acme\Bundle\AppBundle\Model;
+
+use Acme\Bundle\AppBundle\Entity\Color;
+use Pim\Component\Catalog\Model\AbstractProductValue;
+use Pim\Component\Catalog\Model\AttributeInterface;
+
+/**
+ * Color product value for "color" reference data. It's a new many to one relationship
+ *
+ * @author    Julien Janvier <jjanvier@akeneo.com>
+ * @copyright 2015 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class ColorProductValue extends AbstractProductValue
```

Comment:
do we need those values with the referencedata value you introduced?

seems no, which is even more awesome :)